scripts/csi_test/csi_params

```diff
@@ -1,3 +1,3 @@
-#SpaceEfficiency: Compression
+#SpaceEfficiency: Thin
```

Comment:
does this have any effect?

Reply:
its mandatory for testing on svc. in the other hand, cant see what the `SpaceEfficiency: Compression` for

Reply:
I don't think this has any effect.
either undo it or make more changes to make it affective.

Reply:
done